modular_skyrat/modules/connecting_computer/code/game/machinery/computer/buildandrepair.dm

```diff
@@ -0,0 +1,154 @@
+/obj/structure/frame/computer/attackby(obj/item/P, mob/living/user, params)
```

Comment:
You don't need to do this, just go into the actual file, and add your changes, with a //SKYRAT EDIT CHANGE